src/__tests__/User/Login/LoginScreen.test.js

```diff
@@ -60,74 +61,47 @@ describe('LoginScreen', () => {
     })
   })
 
-  describe('when email and password are correct', () => {
-    it('should redirect user to /', () => {
-      const email = 'sjlarson92@gmail.com'
-      const password = '123'
-      wrapper
-        .find({ 'data-testid': 'emailInput' })
-        .simulate('change', { target: { value: email } })
-      wrapper
-        .find({ 'data-testid': 'passwordInput' })
-        .simulate('change', { target: { value: password } })
-      wrapper.find({ 'data-testid': 'loginButton' }).simulate('click')
-      expect(history.push).toHaveBeenCalledWith('/')
+  describe('errorMessage', () => {
+    describe('when there is an errorMessage', () => {
+      it('should render errorMessage', () => {
+        const errorMessage = 'error message'
+        useSelector.mockReturnValue(errorMessage)
+        const newWrapper = shallow(<LoginScreen />)
+        expect(
+          newWrapper.find({ 'data-testid': 'errorMessage' }).text(),
+        ).toEqual('error message')
+      })
     })
-
-    it('should call dispatch with correct params', () => {
-      const email = 'sjlarson92@gmail.com'
-      const password = '123'
-      wrapper
-        .find({ 'data-testid': 'emailInput' })
-        .simulate('change', { target: { value: email } })
-      wrapper
-        .find({ 'data-testid': 'passwordInput' })
-        .simulate('change', { target: { value: password } })
-      wrapper.find({ 'data-testid': 'loginButton' }).simulate('click')
-      expect(dispatch).toHaveBeenCalledWith({ type: TYPES.LOGIN })
+    describe('when there is not an errorMessage', () => {
+      it('should not render errorMessage', () => {
+        useSelector.mockReturnValue('')
+        const newWrapper = shallow(<LoginScreen />)
```

Comment:
when checking that an errorMessage does not exist, you should set it to null or undefined. Empty string is essentially the same as a full string

Reply:
fixed